extensions/roc-plugin-dredd/src/config/roc.config.js

```diff
@@ -1 +1,16 @@
-export default {};
+export default {
+    settings: {
+        test: {
+            dredd: {
```

Comment:
I've rewritten logging in the plugin to reflect this input. We now only rely on the `verbose` flag. I've removed all options except `path`.